feedback/static/html/studio_view.html

```diff
@@ -0,0 +1,102 @@
+<div class="wrapper-comp-settings is-active editor-with-buttons" id="settings-tab">
+  <ul class="list-input settings-list">
+    <li class="field comp-setting-entry is-set"> This XBlock allows
+      you to collect student feedback on pieces of the course. This
+      may be helpful either for course improvement, or to give
+      students a chance to reflect on what they have done.</li>
+  
+    <li class="field comp-setting-entry is-set"> Prior to using this
+      block, we recommend reading about best practices in the edX
+      documentation. In particular, we do recommend asking for
+      structured, specific feedback, and we advise using this in
+      conjunction with
+      the <a href="http://edx.readthedocs.org/projects/edx-partner-course-staff/en/latest/content_experiments/content_experiments_configure.html">RCT
+      framework</a> for matrix sampling.</li>
+
+    <li class="field comp-setting-entry is-set">
+      <div class="wrapper-comp-setting">
+        <label class="label setting-label" for="freeform">Freeform prompt</label>
```

Comment:
If you nest the input within the label, you can get rid of the "for" and the need for ids.
https://www.w3.org/TR/html401/interact/forms.html#edef-LABEL

Using ids is problematic because they must be unique within the page. This is particularly serious in the LMS views, as one could have 2 instances of the feedback xblock in the same page.